src/core/unit_tests/EspressoSystemStandAlone_test.cpp

```diff
@@ -245,12 +265,8 @@ BOOST_FIXTURE_TEST_CASE(espresso_system_stand_alone, ParticleFactory,
     set_particle_q(pid2, -1.);
 
     // set up P3M
-    auto const prefactor = 2.0;
-    auto const mesh = std::vector<int>{8, 8, 8};
-    Coulomb::set_prefactor(prefactor);
-    p3m_set_eps(0.0);
-    p3m_set_params(3.5, mesh.data(), 5, 0.654, 1e-3);
-    p3m_set_mesh_offset(0.5, 0.5, 0.5);
+    mpi_set_tuned_p3m();
+    auto const prefactor = 2.;
```

Comment:
I'm not to sure if it is a good idea to rely on this variable to be the same as the hard-coded one above in
`auto solver = std::make_shared<CoulombP3M>(std::move(p3m), 2., 1, false);`

Reply:
good point, this is fixed in the latest commit